### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by RobbeSneyders**
This PR updates our README to reflect the latest state.

- Update central messaging
- Remove detailed component overview in favor of higher level overview and referral to component hub
- Make example pipelines more visible
- Put contributing options back into README and highlight only most useful ones
- Remove outdated roadmap

### CODE REVIEW

File: docs/contributing.md [hunk 1]
```diff
@@ -1,15 +0,0 @@
-# 👭 Contributing
```
Comment (mrchtr) [inline docs/contributing.md:1]: Do we really want remove this page?

File: docs/contributing.md [hunk 2]
```diff
@@ -1,15 +0,0 @@
...
-| **Framework code contributions** | If you want to help with the development of the Fondant framework, have a look at the issues marked with the [good first issue](https://github.com/ml6team/fondant/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22) label. If you want to add additional functionality, please submit an issue for it first. |
-| **Reusable components**          | Extending our library of reusable components is a great way to contribute. If you built a component which would be useful for other users, please submit a PR adding them to the [components/](https://github.com/ml6team/fondant/tree/main/components) directory. You can find a list of possible contributable components [here](https://github.com/ml6team/fondant/issues?q=is%3Aissue+is%3Aopen+label%3A%22Component+Contribution%22) or your own ideas are also welcome!                                            |
-| **Example pipelines**            | If you built a pipeline with Fondant which can serve as an example to other users, please submit a PR adding them to the [examples/](https://github.com/ml6team/fondant/tree/main/examples) directory.
```
Comment (mrchtr) [inline docs/contributing.md:11]: If we keep the contribution of example pipelines, we could reformulate it that people should open a PR and modify the list of examples. They could add their example to the list.

File: docs/contributing.md (hunk 1 above)
Comment (RobbeSneyders) [reply docs/contributing.md:1]: I included it in the README again in a shortened format. That will be visible on the docs website as well.

File: docs/contributing.md (hunk 2 above)
Comment (RobbeSneyders) [reply docs/contributing.md:11]: I would focus on reusable component contributions for now. I think that one is the most accessible and useful.

File: README.md [hunk 3]
```diff
@@ -296,44 +236,28 @@ fondant <subcommand> --help
...
-[our discord](https://discord.gg/HnTdWhydGp) or submit an idea in our
-[Github Discussions](https://github.com/ml6team/fondant/discussions/categories/ideas).
+|                                  |                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |
+|----------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
```
Comment (PhilippeMoussalli) [inline README.md:244]: I think there is an extra hidden row that can be removed

**Review by PhilippeMoussalli — APPROVED** [review]
Thanks Robbe! Much cleaner

**Review by mrchtr — APPROVED** [review]
> I would focus on reusable component contributions for now. I think that one is the most accessible and useful.

Alright, makes sense. Thanks!

File: README.md (hunk 3 above)
Comment (RobbeSneyders) [reply README.md:244]: This is the empty header. If we remove it, it will no longer render as a table.

**Pull request merged**